stories/overlays/Modal.stories.js

```diff
@@ -53,12 +53,8 @@ storiesOf('Overlays|Modal', module)
     )
   })
 
-  .add('Toggle Example', withInfo({
```

Comment:
I disabled this initially because there was some conflict while using addon-info with the old story format and new at the same time, but I think I can enable this again